### PR TITLE
working with drag and select

### PR DESCRIPTION
**Pull request by JupitersLB**
@Naokimi needs to be pulled and checked just incase.

### CODE REVIEW

File: app/controllers/api/v1/user_availabilities_controller.rb [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-  end
+  # def create
+  #   @user_availability = UserAvailability.new(availability_params)
+  #   @user_availability.user = current_user
+  #   if @user_availability.save
+  #     render json: @user_availability
+  #   else
+  #     render_error
+  #   end
+  # end
```
Comment (Naokimi) [inline app/controllers/api/v1/user_availabilities_controller.rb]: if we are not going to use it anymore better delete the code and the route for readability sake

File: app/controllers/api/v1/user_availabilities_controller.rb [hunk 2]
```diff
@@ -33,8 +35,8 @@ def availability_params
     params.require(:user_availability).permit(:time_block_id, :recurring)
   end
 
-  def time_block_ids_params
-    params.require(:time_block_ids)
+  def multi_availability_params
+    params.require(:user_availability).permit(:recurring, :time_block_ids  => [])
```
Comment (Naokimi) [inline app/controllers/api/v1/user_availabilities_controller.rb]: `time_block_ids : []` doesn't work?

Comment (JupitersLB) [reply app/controllers/api/v1/user_availabilities_controller.rb]: I can try it

**Comment by JupitersLB** [comment]
@Naokimi updated

**Review by Naokimi — APPROVED** [review]

**Pull request merged**